source/include/ExecutionContext.h

```diff
@@ -89,8 +90,9 @@ class ExecutionContext
     ~ExecutionContext();
  private:
     ECONTEXT    VIClumpQueue    _runQueue;         // Clumps ready to run
+    ECONTEXT    VIClumpQueue    _waitQueue;
     ECONTEXT    Int32           _breakoutCount;   // Inner execution loop "breaks out" when this gets to 0
-
+    ECONTEXT    Int32          _viPaused = 1;
```

Comment:
Makes this an enum instead so that it's easier to follow what the state is.

Reply:
Also the variable should be called _executionState since it is not specific to a single VI but the execution context as a whole.